src/OrbitGl/App.cpp

```diff
@@ -2269,6 +2270,46 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::UpdateProcessAndModuleList()
       });
 }
 
+void OrbitApp::LoadAllSymbols() {
+  std::vector<const ModuleData*> all_modules = module_manager_->GetAllModuleData();
+
+  auto ggpvlk_it = std::find_if(
+      all_modules.begin(), all_modules.end(),
+      [](const ModuleData* module) { return absl::StrContains(module->file_path(), "ggpvlk.so"); });
+  if (ggpvlk_it != all_modules.end()) {
+    const ModuleData* ggpvlk_module{*ggpvlk_it};
+    if (!ggpvlk_module->is_loaded()) {
+      std::ignore = RetrieveModuleAndLoadSymbols(ggpvlk_module);
+      all_modules.erase(ggpvlk_it);
+    }
+  }
+
+  const ProcessData* process = GetTargetProcess();
+  if (process == nullptr) {
+    // Orbit is not currently connected, so this uses the process from capture data, which then is
+    // from the capture that was loaded from file.
+    process = GetCaptureData().process();
+  }
+  ORBIT_CHECK(process != nullptr);
```

Comment:
I think this could also be a separate function

Reply:
done